src/components/playerstats/playerstats.js

```diff
@@ -200,7 +200,7 @@ function getDisplayTranscodeFps(session, player) {
     const mediaSource = playbackManager.currentMediaSource(player) || {};
     const videoStream = (mediaSource.MediaStreams || []).find((s) => s.Type === 'Video') || {};
 
-    const originalFramerate = videoStream.AverageFrameRate;
+    const originalFramerate = videoStream.AverageFrameRate || videoStream.RealFrameRate;
```

Comment:
Better to use `videoStream.ReferenceFrameRate || videoStream.RealFrameRate`?

See also https://github.com/jellyfin/jellyfin/pull/12603

Reply:
Ah, I have mostly only looked at jellybin-web, not at jellyfin-server. 

I assume that ReferenceFrameRate could only possibly exist with newer Jellyfin-Server releases. Otherwise, if you have a newer web but older server then nothing would change.

I guess it depends on if web releases should work with older server releases or not. If yes then maybe it is needed to do
`videoStream.ReferenceFrameRate || videoStream.AverageFrameRate || videoStream.RealFrameRate`
But if older server releases dont matter, then I currently see no way how ReferenceFrameRate could ever be null, in that case maybe even `const originalFramerate = videoStream.ReferenceFrameRate` is enough :thinking: 

Though `videoStream.ReferenceFrameRate || videoStream.RealFrameRate` is probably also a healthy middleground. Something that ensures backwards compatibility (without fixing the bug) but something that also works for the future :thinking:

Reply:
We try to support one "major" version back when possible.